src/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import HomePage from './components/homePage/HomePage';
+import MyAppointments from './components/appointmentInfo/MyAppointments';
+import DoctorList from './components/doctorInfo/DoctorList';
+import DeleteDoctor from './components/doctorInfo/DeleteDoctor';
+import PatientList from './components/forSuperUser/PatientList';
+import Appointment from './components/appointmentInfo/Appointment';
+import AppointmentList from './components/forSuperUser/AppointmentList';
+import UserList from './components/forSuperUser/UserList';
+import DoctorDetails from './components/doctorInfo/DoctorDetails';
+import ModalAppoint from './components/appointmentInfo/ModalAppoint';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:


![image](https://github.com/Tracy-miranja/appoint-doctor-frontend/assets/57421823/3c083515-2fd3-46e5-a022-b01ce2df9240)

That way, your code will meet the latest standards. 💪

Reply:
Thanks @Meltrust for the review. this one is resolved